oslo_messaging/metrics.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import threading, queue
+import statsd
+
+LOG = logging.getLogger(__name__)
+
+class Metrics():
```

Comment:
Nova is using this code with `python2` still. Therefore, I would assume this has to be `class Metrics(object):` to make it a new-style object.